lighthouse-cli/test/smokehouse/dobetterweb/dbw-expectations.js

```diff
@@ -117,7 +117,7 @@ module.exports = [
         score: false,
         extendedInfo: {
           value: {
-            length: 4
+            length: 5
```

Comment:
The xhr: `xhr.open('GET', 'dbw_tester.html', false);`. Moved from using an http2 endpoint (lighthouse-viewer.appspot.com).